orchestra/migrations/0091_auto_20201023_0912.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.13 on 2020-10-23 09:12
```

Comment:
Rename the migration file

Reply:
Make sure that the problem has already been solved. We already have other places where we serialize/deserialize json-fields

Reply:
Couldn't find usage examples where we build querysets using nested fields of a JSONField, e.g. `queryset.filter(some_json_field__nested_field='value')`